recipes/clickhouse-cpp/all/conanfile.py

```diff
@@ -0,0 +1,129 @@
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeToolchain,CMakeDeps, cmake_layout
+from conan.tools.files import copy, get
+from conan.tools.build import check_min_cppstd
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.scm import Version
+import os
+
+required_conan_version = ">=1.51.3"
+
+class ClickHouseCppConan(ConanFile):
+    name = "clickhouse-cpp"
+    homepage = "https://github.com/ClickHouse/clickhouse-cpp"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "ClickHouse C++ API"
+    license = "Apache-2.0"
+    topics = ("database", "db", "clickhouse")
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "build_bench": [True, False],
```

Comment:
```suggestion
        "enable_benchmark": [True, False],
```
Not really required, but a good recommendation: https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/conanfile_attributes.md#predefined-options-and-known-defaults